libraries/rush-lib/src/api/Subspace.ts

```diff
@@ -20,6 +20,7 @@ export interface ISubspaceOptions {
   subspaceName: string;
   rushConfiguration: RushConfiguration;
   splitWorkspaceCompatibility: boolean;
+  preventFilteredInstall: boolean;
```

Comment:
```suggestion
  preventFilteredInstall: boolean | undefined;
```

Then you can get rid of the `!!` when setting this property.

Reply:
Wouldn't this just move the `!!` to the subspace constructor? Or do you mean this property should be `boolean | undefined` everywhere?